conf/defaults.ini

```diff
@@ -931,6 +931,10 @@ max_annotations_to_keep =
 # Configures the batch size for the annotation clean-up job. This setting is used for dashboard, API, and alert annotations.
 cleanupjob_batchsize = 100
 
+# Enforces the maximum allowed length of the tags for any newly introduced annotations. It can be between 1 and 4096 inclusive (which is the respective's column length). Default value is 500.
```

Comment:
Isn't 0 also possible?

Reply:
right now it is but maybe it does not make much sense; I think that we have to allow only values between (500, 4096). WDYT?